profile/index.php

```diff
@@ -8,205 +8,214 @@
+				<!-- TemplateBeginEditable name="pagetitle" -->
+				<h1>Profile</h1>
+				<!-- TemplateEndEditable -->
+			</header>
+			<!-- TemplateBeginEditable name="herogroup1" -->
+			<!-- TemplateEndEditable -->
 		</div>
-		<div class="wdn-menu-trigger wdn-content-slide">
-			<label for="wdn_menu_toggle" class="wdn-icon-menu">Menu</label>
-			<?php wdnInclude("/wdn/templates_4.1/includes/share.html"); ?>
+		<!-- TemplateBeginEditable name="herogroup2" -->
+		<div class="dcf-hero-group-2">
 		</div>
-		<main id="wdn_content_wrapper" role="main" class="wdn-content-slide" tabindex="-1">
-			<div id="maincontent" class="wdn-main">
-				<div id="pagetitle">
-					<!-- TemplateBeginEditable name="pagetitle" -->
-					<h1>Profile</h1>
-					<!-- TemplateEndEditable -->
-				</div>
-				<!-- TemplateBeginEditable name="maincontentarea" -->
-
-				<!-- Personal Information -->
-				<div class="wdn-band wdn-light-neutral-band">
-					<div class="wdn-inner-wrapper wdn-inner-padding-sm">
-						<div class="personal-info">
-							<h2 class="clear-top inline">Personal Information</h2>
-							<button class="wdn-button wdn-pull-right clear-top" id="personalInformationEditButton">Edit</button>
-
-							<div>
-								<label for="firstName" id="firstNameLabel">First Name:</label>
-								<span id="firstNameText"></span>
-								<input type="text" id="firstNameInput" style="display:none;" />
-							</div>
-
-							<div>
-								<label for="lastName" id="lastNameLabel">Last Name:</label>
-								<span id="lastNameText"></span>
-								<input type="text" id="lastNameInput" style="display:none;" />
-							</div>
-
-							<div>
-								<label for="email" id="emailLabel">Email:</label>
-								<span id="emailText"></span>
-								<input type="text" id="emailInput" style="display:none;" disabled />
-							</div>
-
-							<div>
-								<label for="phoneNumber" id="phoneNumberLabel">Phone Number:</label>
-								<span id="phoneNumberText"></span>
-								<input type="text" id="phoneNumberInput" style="display:none;" />
-							</div>
+		<!-- TemplateEndEditable -->
+	</div>
+	<div class="dcf-main-content dcf-wrapper">
+		<!-- TemplateBeginEditable name="maincontentarea" -->
+		<!-- Personal Information -->
+		<div class="dcf-bleed unl-bg-lightest-gray">
+			<div class="dcf-wrapper wdn-inner-padding-sm">
+				<div class="personal-info">
+					<h2 class="inline">Personal Information</h2>
+					<button class="dcf-btn dcf-float-right" id="personalInformationEditButton">Edit</button>
 
-							<button class="wdn-button wdn-button-triad" id="personalInformationSaveButton" style="display:none;">Save</button>
-							<button class="wdn-button wdn-button-brand" id="personalInformationCancelButton" style="display:none;">Cancel</button>
-						</div>
+					<div>
+						<label for="firstName" id="firstNameLabel">First Name:</label>
+						<span id="firstNameText"></span>
+						<input type="text" id="firstNameInput" style="display:none;" />
```

Comment:
These currently have `style="display:none;"`, should we be putting `class="dcf-txt-input"` or whatever it is, on these?

Reply:
I can't find any documentation for a text input class. Closing as no-op